.travis.yml

```diff
@@ -11,7 +11,7 @@ dist: xenial
 
 # Defaults
 language: node_js
-node_js: 'lts/*'
```

Comment:
Due to the wildcard in `lts/*` we've actually been running node 12 in CI for the last couple of days (node 12 entered LTS on 21st). Switching explicitly to a numbered version